pages/learn/manage/device-statuses.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Device statuses
+excerpt: Determine the status and connectivity of a device
+---
+
+# Device statuses
+
+The application overview page shows the status of each device in the application.
+
+<img src="/img/common/main_dashboard/application_device_status.png" alt="Application device status" width="40%" >
+
+Each device can have one of the following statuses:
+
+| Status            | Description                                                                                  |
+|-------------------|----------------------------------------------------------------------------------------------|
+| Online            | The device is online and is connected to either the VPN or has recent API communications.    |
+| Configuring       | The device is applying a configuration update.                                               |
```

Comment:
More context: Devices show as configuring during the provisioning process for flasher device types, and during host os updates (please give this a try).

See: https://github.com/balena-os/meta-balena/blob/006bc19161053147f47fa12e4cc063da5011215a/meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher#L144
See: https://github.com/balena-os/resin-os-misc/blob/7f0edcb78ed191465b5686086c0ab0260ec8421d/resinhup/run-resinhup.sh
See: https://github.com/balena-io/balena-proxy/blob/79c9b2d6f0c90bd6945e0ed7a65d93b87c1fdbc0/src/services/actions-backend/actions/scripts/upgrade-1.x.sh